stx-rosetta.Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN apt-get -y update && apt-get -y install openjdk-11-jre-headless
 WORKDIR /app
 
 RUN git clone -b $API_TAG --depth 1 https://github.com/blockstack/stacks-blockchain-api.git .
-COPY ./rosetta-constants.ts /app/src/api/
+# COPY ./rosetta-constants.ts /app/src/api/
 RUN grep process.env.STACKS_NETWORK /app/src/api/rosetta-constants.ts
```

Comment:
Getting an error from this line when building the dockerfile standalone 
```
Step 6/63 : RUN grep process.env.STACKS_NETWORK /app/src/api/rosetta-constants.ts
 ---> Running in 045d16eb19c2
The command '/bin/sh -c grep process.env.STACKS_NETWORK /app/src/api/rosetta-constants.ts' returned a non-zero code: 1
```
If I comment out this line, everything seems to work as expected. Should these lines, 11-12, be removed?